src/Rector/Deprecation/NodeLoadMultipleRector.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation;
+
+use DrupalRector\Rector\Deprecation\Base\EntityLoadBase;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * Replaced deprecated node_load_multiple() calls.
+ *
+ * See https://www.drupal.org/node/2266845 for change record.
+ *
+ * What is covered:
+ * - See EntityLoadBase.php
+ *
+ * Improvement opportunities
+ * - See EntityLoadBase.php
+ */
+final class NodeLoadMultipleRector extends EntityLoadBase
```

Comment:
This seems like it will duplicate / be redundant to the implementation of https://github.com/palantirnet/drupal-rector/blob/master/src/Rector/Deprecation/NodeLoadRector.php.